src/vtt-segment-loader.js

```diff
@@ -476,11 +476,12 @@ export default class VTTSegmentLoader extends SegmentLoader {
 
     const timestampmap = segmentInfo.timestampmap;
     const diff = (timestampmap.MPEGTS / ONE_SECOND_IN_TS) - timestampmap.LOCAL + mappingObj.mapping;
-
+    const mpegTsRollover = Math.floor((2**33) / ONE_SECOND_IN_TS);
```

Comment:
I wonder if there's a way to know whether TS segments are used here and only apply this rollover in those cases. @gesinger thoughts?